web/src/main/webapp/WEB-INF/data/data/formatter/xslt/render-layout.xsl

```diff
@@ -94,54 +94,48 @@
                                    select="."/>
             </xsl:for-each>
           </div>
-          <div class="gn-md-side col-md-4">
+          <div class="gn-md-side gn-md-side-advanced col-md-4">
             <xsl:apply-templates mode="getOverviews" select="$metadata"/>
 
-            <br/>
-
-            <section>
+            <section class="gn-md-side-providedby">
               <h4>
-                <i class="fa fa-fw fa-cog">&#160;</i>&#160;
+                <i class="fa fa-fw fa-cog"></i>
```

Comment:
I dont think you should remove those. It will generate wrong HTML for chrome as far as I remember no ?

Reply:
I don't know, but if that's the case I can add them again.

Reply:
The issue was that some icons were repeated and that the sidebar was displayed below the main content due to invalid/incomplete HTML I think